nautobot/docs/user-guide/administration/installation/install_system.md

```diff
@@ -20,14 +20,14 @@ This will install:
 
 === "Ubuntu/Debian"
 
-    ```bash
+    ```bash title="Install system dependencies"
     sudo apt update -y
     sudo apt install -y git python3 python3-pip python3-venv python3-dev redis-server
     ```
 
-=== "RHEL8 + Derivatives"
+=== "RHEL8"
 
-    ```bash
+    ```bash title="Install system dependencies"
     sudo dnf check-update
     sudo dnf install -y git python38 python38-devel python38-pip redis
```

Comment:
We should probably update this soon to whatever the latest `python3*` version is that RHEL8 provides. Not blocking.